### DISCUSSION TITLE
new user guidance

### DISCUSSION THREAD

**Discussion by kamaradski** [OpenCCU/OpenCCU#1015] [🇺🇸 Ideas]
Hi, 

I'm a new user of raspberrymatic (using the ELV rpi-rf-mod), and my user-experience so far is similar to walking into a brick wall.

Installing and setting up basic settings was easy enough, however, getting my homematic-ip gear re-setup is not as intuitive as I would have hoped. 

Things like:
- linking my homematic app to the raspberrymatic
- learning-in devices
- linking devices to rooms
- basic automation

these are all things I would expect to be able to figure out rather quickly in just a few hours but instead, I'm walking in circles. a "first-time setup guide" or "frequently performed tasks" or even a more extensive user manual would be highly appreciated to get me going quicker. 

I hope we can use this thread to collect all such guides and information (even links to video's) here for all future new users joining this great project.

**Comment by jens-maus** [comment]
Thanks for this thread. Indeed, RaspberryMatic could do better in terms of documentation and guidance for first-time users.

Thus, if you collect stuff here it would be indeed great. And if you could also start e.g. translating the current documentation (https://github.com/jens-maus/RaspberryMatic/wiki) to english it would be great as well. :)

**Reply by kamaradski** [reply]
i started adding to the wiki, but I cannot get the local push to work. Could you double check if the repo settings would allow me so?

I have tried both the https and the ssh link and both give me a permission error

url=git@github.com/jens-maus/RaspberryMatic.wiki.git
```
$git push
fatal: 'git@github.com/jens-maus/RaspberryMatic.wiki.git' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

or:
```
$git push
Username for 'https://github.com': ***@***.com
Password for 'https://***@***.com@github.com':
remote: Permission to jens-maus/RaspberryMatic.wiki.git denied to kamaradski.
fatal: unable to access 'https://github.com/jens-maus/RaspberryMatic.wiki.git/': The requested URL returned error: 403
```

I would need this in order to put the pictures in the local ".pictures" folder. It's cleaner than hosting them myself.

**Reply by jens-maus** [reply]
Sorry, but I just learned that a direct git push to the wiki git repository only works for people having direct push access to the make repository as well. So please make your changes via the github webpage instead and then afterwards I could change/modify it to your needs directly in the git wiki repository

**Reply by kamaradski** [reply]
ok in that case i will link the screenshots already in the wiki page, and email you the pictures & pathname?

**Reply by jens-maus** [reply]
No, upload them via the webpage, I will then download them and add them to the repo afterwards

**Comment by kamaradski** [comment]
One of the key learnings I had yesterday was that one can read the official eq-3 WebUI documentation as the UI is similar/the same as raspberrymatic: https://www.eq-3.de/Downloads/eq3/download%20bereich/handbuecher/WebUI_Handbuch_eQ-3.pdf (German)

I have never owned an actual CCU since I jumped from the Homematic access point directly to raspberrymatic, so I was not aware of what the CCU UI actually looks like. 

Reading the "WebUI_Handbuch", insta-solved most of my open problems and questions.
